### PR TITLE
kill couchlog

### PR DESCRIPTION
**Pull request by nickpell**
@millerdev @snopoke @czue @dannyroberts

### CODE REVIEW

File: corehq/form_processor/backends/sql/supply.py [hunk 1]
```diff
@@ -12,16 +12,6 @@ def get_or_create_by_location(cls, location):
         sp = SupplyPointSQL.get_by_location(location)
         if not sp:
             sp = make_supply_point(location.domain, location)
-
-            # todo: if you come across this after july 2015 go search couchlog
```
Comment (nickpell) [inline corehq/form_processor/backends/sql/supply.py]: @snopoke Is this ok to remove?

File: corehq/form_processor/backends/couch/supply.py [hunk 2]
```diff
@@ -14,16 +14,6 @@ def get_or_create_by_location(cls, location):
         sp = location.linked_supply_point()
         if not sp:
             sp = make_supply_point(location.domain, location)
-
-            # todo: if you come across this after july 2015 go search couchlog
```
Comment (nickpell) [inline corehq/form_processor/backends/couch/supply.py]: @snopoke And this?

Comment (snopoke) [inline corehq/form_processor/backends/couch/supply.py]: replace with a soft assert?

File: localsettings.example.py [hunk 3]
```diff
@@ -232,7 +232,6 @@
 
 # our production logstash aggregation
 LOGSTASH_DEVICELOG_PORT = 10777
-LOGSTASH_COUCHLOG_PORT = 10888
 LOGSTASH_AUDITCARE_PORT = 10999
 LOGSTASH_HOST = 'localhost'
```
Comment (snopoke) [inline localsettings.example.py]: don't seem like these are used anymore either

**Review by snopoke — APPROVED** [review]

**Comment by nickpell** [comment]
@snopoke 1a2ee9784874bf04d237568ce92e686d64a57586

**Review by czue — APPROVED** [review]
lgtm. any reason to think this needs any QA? (not advocating just asking). 

are there any ansible changes we should also include as a part of this?

**Review by snopoke — APPROVED** [review]

**Comment by nickpell** [comment]
@czue 

> lgtm. any reason to think this needs any QA? (not advocating just asking).

I figured not.  It's on prod now and so far no problems.

> are there any ansible changes we should also include as a part of this?

https://github.com/dimagi/commcarehq-ansible/pull/568

**Comment by czue** [comment]
:+1: thanks

**Pull request merged**